git-server/Dockerfile

```diff
@@ -13,5 +13,33 @@ FROM debian:buster-slim@sha256:c8152821b158dd171b4acf92afb0a58fc2faa179a7e0af8ac
 EXPOSE 8778/tcp
 RUN apt-get update && apt-get install -y libssl1.1 git && rm -rf /var/lib/apt/lists/*
 COPY --from=build /usr/local/cargo/bin/radicle-git-server /usr/local/bin/radicle-git-server
+
+# # create a `git` user & group, creates /home/git
```

Comment:
Not sure if it makes sense to add this dead code or not. Need to discuss ssh access for the git user and security considerations.